Makefile

```diff
@@ -16,6 +16,10 @@ checkstyle: bin/gometalinter
 lint: bin/gometalinter
 	./bin/gometalinter --deadline=240s --vendor ./src/naksu/...
 
+test:
```

Comment:
`test` rule should probably be hooked to normal build chain. At least `docker` target could depend on this, and this could depend on `lint`. `all` should also run tests as first operation